lua/starfall/libs_cl/render.lua

```diff
@@ -857,6 +857,31 @@ function render_library.setColor(clr)
 	surface.SetTextColor(clr)
 end
 
+--- Gets the draw color modulation. Note that, unlike the equivalent GLua function, this function returns numbers in a 0-255 range.
+-- @return number Red channel
+-- @return number Green channel
+-- @return number Blue channel
+function render_library.getColorModulation()
+	if not renderdata.isRendering then SF.Throw("Not in a rendering hook.", 2) end
+		
+	local r, g, b = render.GetColorModulation()
+	return r * 255, g * 255, b * 255
+end
+	
+--- Sets the draw color modulation. Note that, unlike the equivalent GLua function, this function takes numbers in a 0-255 range.
+-- @param number|Color clr Color type, or red channel
+-- @param number? g Optional green channel, if you wish to use r, g, b as arguments instead of a Color
+-- @param number? b Optional blue channel, read above
+function render_library.setColorModulation(clr, g, b)
```

Comment:
This needs to be reset in `cleanupRender`, otherwise can affect other stuff.
Can probably just call `render.SetColorModulation(1, 1, 1)`, no need to remember the state from `prepareRender`.